packages/dd-trace/src/payload-tagging/config/aws.json

```diff
@@ -0,0 +1,30 @@
+{
+  "sns": {
+    "request": [
+      "$.Attributes.KmsMasterKeyId",
+      "$.Attributes.PlatformCredential",
+      "$.Attributes.PlatformPrincipal",
+      "$.Attributes.Token",
+      "$.AWSAccountId",
+      "$.Endpoint",
+      "$.OneTimePassword",
+      "$.phoneNumber",
+      "$.PhoneNumber",
+      "$.Token"
+    ],
+    "response": [
+      "$.Attributes.KmsMasterKeyId",
+      "$.Attributes.Token",
+      "$.Endpoints.*.Token",
+      "$.PhoneNumber",
+      "$.PhoneNumbers",
+      "$.phoneNumbers",
+      "$.PlatformApplication.*.PlatformCredential",
+      "$.PlatformApplication.*.PlatformPrincipal",
+      "$.Subscriptions.*.Endpoint"
+    ],
+    "expand": [
+      "$.MessageAttributes.*.StringValue"
```

Comment:
`$.MessageBody` should also be in this list.

Reply:
SNS only:
`$.MessageAttributes._datadog.StringValue` = base64
`$.MessageAttributes.*.StringValue` = json

Reply:
Actually just check DataType. SNS can have a DataType=binary, others will just have strings. And _datadog will be JSON.

Reply:
does expansion rules apply to both request and response?